src/SB/Game/zGrid.h

```diff
@@ -5,6 +5,9 @@
 
 #include "xEnt.h"
 #include "xGrid.h"
+#include "xMath.h"
+
+#include <PowerPC_EABI_Support\MSL_C\MSL_Common\cmath>
```

Comment:
Additionally, I might be wrong so ignore this if it doesn't work but I"m pretty sure the cmath include should just be this:
```suggestion
#include "xMath.h"

#include <cmath>
```

Reply:
Nope, full path is necessary unfortunately.

Reply:
Ok, this may be a project configuration issue I should look into.

Reply:
It should still be  moved to zGrid.cpp though